backend/firebaseFunctions.js

```diff
@@ -125,7 +173,7 @@ export async function loginUser(email, password) {
 }
 
 export async function checkUsername(username) {
-	let usernames = (await readData(`usernames`)) || {};
+	let usernames = await readData(`usernames`);
```

Comment:
```suggestion
	let usernames = (await readData(`usernames`)) || {};
```
this will help incase something goes wrong with the DB and all the users are deleted or inaccessible